### PR TITLE
something by elif

### PR DESCRIPTION
**Pull request by kaynakef**
Hello

### CODE REVIEW

**Comment by nglady21** [comment]
hey

**Comment by LauraSchalk** [comment]
Hello! Great job creating a pull request!

**Comment by Altynai07** [comment]
hello everyone

File: src/Test.java [hunk 1]
```diff
@@ -2,5 +2,6 @@ public class Test {
     public static void main(String[] args) {
         System.out.println("This is batch12 practice");
         System.out.println("This is batch12 2nd practice");
+        System.out.println("something from elif");
```
Comment (Renas63) [inline src/Test.java:5]: Maybe you can start with upper case

**Comment by Zhaziko** [comment]
I'm finally here

**Comment by kaynakef** [comment]
updated

**Comment by mroxana3** [comment]
im trying

**Review by Renas63 — APPROVED** [review]

**Comment by Aiike95** [comment]
hi there

**Comment by hacerturan** [comment]
Hi

**Pull request merged**